src/geocat/comp/__init__.py

```
@@ -1105,4 +1105,248 @@ def rcm2points(lat2d, lon2d, fi, lat1dPoints, lon1dPoints, opt=0, msg=None, meta
+
+    # Parsing Options
+    options = {}
+    if "method" in kwargs:
+        if not isinstance(kwargs["method"], int):
+            raise TypeError('ERROR triple2grid: `method` arg must be an integer. Set it to either 1 or 0.')
+        input_method = np.asarray(kwargs["method"]).astype(np.int_)
+        if (input_method != 0) and (input_method != 1):
+            raise TypeError('ERROR triple2grid: `method` arg accepts either 0 or 1.')
+        options[b'method'] = input_method
+
+        # `distmx` is only applicable when `method`==1
+        if input_method:
+            if "distmx" in kwargs:
+                input_distmx = np.asarray(kwargs["distmx"]).astype(np.float_)
+                if input_distmx.size != 1:
+                    raise ValueError("ERROR triple2grid: Provide a scalar value for `distmx` !")
+                options[b'distmx'] = input_distmx
+
+    if "domain" in kwargs:
+        input_domain = np.asarray(kwargs["domain"]).astype(np.float_)
+        if input_domain.size != 1:
+            raise ValueError("ERROR triple2grid: Provide a scalar value for `domain` !")
+        options[b'domain'] = input_domain
+
+    msg = kwargs.get("msg", np.nan)
+    meta = kwargs.get("meta", False)
+
+    # the input arguments must be convertible to numpy array
+    if isinstance(x, xr.DataArray):
+        x = x.values
+    if isinstance(y, xr.DataArray):
+        y = y.values
+    if isinstance(data, xr.DataArray):
+        data = data.values
+    if isinstance(xgrid, xr.DataArray):
+        xgrid = xgrid.values
+    if isinstance(ygrid, xr.DataArray):
+        ygrid = ygrid.values
+
+    if isinstance(data, np.ndarray):
+        fo = _ncomp._triple2grid(x, y, data, xgrid, ygrid, options, msg)
+    else:
+        raise TypeError
+
+    if meta and isinstance(input, xr.DataArray):
+        pass	 # TODO: Retaining possible metadata might be revised in the future
```

Rather than silently ignoring this option it would probably be better to throw an error, or remove it. What do others think?

---

It would be logical to throw an exception to have an informed feedback that NCL `_wrap()` functions are not yet supported.

---

https://github.com/NCAR/geocat-comp/pull/53/commits/0d8d4f53013fd903ee03df52ace649a58ea354fa